Formula/hdf5.rb

```diff
@@ -0,0 +1,85 @@
+class Hdf5 < Formula
+  desc "File format designed to store large amounts of data"
+  homepage "https://www.hdfgroup.org/HDF5"
+  url "https://www.hdfgroup.org/ftp/HDF5/releases/hdf5-1.10/hdf5-1.10.1/src/hdf5-1.10.1.tar.bz2"
+  sha256 "9c5ce1e33d2463fb1a42dd04daacbc22104e57676e2204e3d66b1ef54b88ebf2"
+
+  bottle do
+    sha256 "cfcd0e27ff297c39b495285fc623a34b1751bdb4993beab94d86f52931bc3448" => :sierra
+    sha256 "a4e677a58a57aaaa4be1dd7fd165b15272ecbf265b7acfa7b89952aea3460467" => :el_capitan
+    sha256 "d9a15be0e5f7091084db292e32a469f9eaff7e3615b429c65f7fde2ef230a8f7" => :yosemite
+    sha256 "a9b8f04e32dba85a013f6c1be017e336e5452091e5e29f27a2d6bc2ba965dbc5" => :x86_64_linux
+  end
+
+  deprecated_option "enable-fortran" => "with-fortran"
+  deprecated_option "enable-parallel" => "with-mpi"
+  deprecated_option "enable-cxx" => "with-cxx"
+
+  option "with-mpi", "Compile with parallel support"
+  option :cxx11
+
+  depends_on "szip"
+
+  depends_on :fortran => :optional
```

Comment:
This should move down, according to build before non-optional, and non-optional before optional.